syncd/syncd_flex_counter.cpp

```diff
@@ -341,7 +341,11 @@ bool FlexCounter::isCounterMapsEmpty()
 {
     SWSS_LOG_ENTER();
 
-    return m_queueCounterIdsMap.empty() && m_portCounterIdsMap.empty() && m_queueAttrIdsMap.empty();
+    return m_queueCounterIdsMap.empty() &&
+           m_portCounterIdsMap.empty() &&
+           m_queueAttrIdsMap.empty() &&
+           m_queuePlugins.empty() &&
+           m_portPlugins.empty();
```

Comment:
the function name does not quite match now. also if you look line 326, line 309 in this file, it checks m_queuePlugins/m_portPlugins, are those duplicated?

Reply:
Done